### ISSUE TITLE
The afl crash dir and qsym error dir.

### ISSUE THREAD

**Issue by zhanggenex** [sslab-gatech/qsym#68]
Hi, I am confused with the the files in afl **crahes** and qsym **errors**. Are those files duplicate or different? Thanks.

**Comment by jakkdu** [comment]
Hi. afl's **crashes** directory and qsym's **error** directory contain test cases that crash afl-instrumented binary and qsym (+ binary), respectively. 
Since afl's instrumentation is very simple, crash files  in afl's **crashes** directory show bugs in a target program.

However, qsym's mechanism is much more complicated than afl's one and thus is error-prone.
qsym only receives a test case from afl, which does not crash afl-instrumented binary.
Therefore, if a certain test case crashes qsym, which does not crash afl, it often means that there is a bug in qsym.
Thus, qsym's **errors** directory show bugs in qsym itself, which is useful for debugging.

In summary:
afl's **crashes** directory: contains test cases that show bugs in a program
qsym's **error** directory: contains test cases that show bugs in qsym itself.

Thanks.

**Comment by zhanggenex** [comment]
@jakkdu Thanks!!!
